### PR TITLE
Add libcdio

### PR DESCRIPTION
**Pull request by asdfugil**
udf, iso9660, cddb, cd-rom, iso images libraries and utilities.

### CODE REVIEW

File: build_info/libcdio-dev.control [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Version: @DEB_LIBCDIO_V@
+Depends: libcdio19 (= @DEB_LIBCDIO_V@)
+Section: Development
+Priority: optional
+Description: development files for libcdio
+ This library is to encapsulate CD-ROM reading and control. Applications wishing to be oblivious of the OS- and device-dependant properties of a CD-ROM can use this library.
+ .
+ Some support for disk image types like BIN/CUE and NRG is available, so applications that use this library also have the ability to read disc images as though they were CD's.
+ .
+ This package contains the header files and static library needed to compile applications that use libcdio.
```
Comment (TheRealKeto) [inline build_info/libcdio-dev.control:13]: So many damn control files....

```suggestion
Description: development files for libcdio.
 This library is to encapsulate CD-ROM reading and control. Applications
 wishing to be oblivious of the OS- and device-dependant properties of a
 CD-ROM can use this library. Some support for disk image types like BIN/CUE
 and NRG is available, so applications that use this library also have the ability
 to read disc images as though they were CD's. This package contains the header
 files and static library needed to compile applications that use libcdio.
```

File: build_info/libcdio++1.control [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Version: @DEB_LIBCDIO_V@
+Section: Libraries
+Priority: optional
+Depends: libcdio19 (>= @DEB_LIBCDIO_V@)
+Description: C++ library to read and control CD-ROM
+ This C++ library is to encapsulate CD-ROM reading and control. Applications wishing to be oblivious of the OS- and device-dependant properties of a CD-ROM can use this library.
+ .
+ Some support for disk image types like BIN/CUE and NRG is available, so applications that use this library also have the ability to read disc images as though they were CD's.
```
Comment (TheRealKeto) [inline build_info/libcdio++1.control:11]: Why does this need to be debianized into so many packages.....

```suggestion
Description: C++ library to read and control CD-ROM.
 This C++ library is to encapsulate CD-ROM reading and control. Applications
 wishing to be oblivious of the OS- and device-dependant properties of a CD-ROM
 can use this library. Some support for disk image types like BIN/CUE and NRG is available,
 so applications that use this library also have the ability to read disc images as though
 they were CD's.
```

File: build_info/libcdio++-dev.control [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Version: @DEB_LIBCDIO_V@
+Depends: libcdio++1 (= @DEB_LIBCDIO_V@), libcdio-dev (= @DEB_LIBCDIO_V@)
+Section: Development
+Priority: optional
+Description: C++ library to read and control CD-ROM (development files)
+ This package contains development files (headers and static library) for the libcdio++ library.
+ .
+ This library is to encapsulate CD-ROM reading and control. Applications wishing to be oblivious of the OS- and device-dependant properties of a CD-ROM can use this library.
+ .
+ Some support for disk image types like BIN/CUE and NRG is available, so applications that use this library also have the ability to read disc images as though they were CD's.
```
Comment (TheRealKeto) [inline build_info/libcdio++-dev.control:13]: Please stop with all of these control files......

```suggestion
Description: C++ library to read and control CD-ROM (development files).
 This package contains development files (headers and static library) for the
 libcdio++ library. This library is to encapsulate CD-ROM reading and control.
 Applications wishing to be oblivious of the OS- and device-dependant properties
 of a CD-ROM can use this library. Some support for disk image types like BIN/CUE
 and NRG is available, so applications that use this library also have the ability
 to read disc images as though they were CD's.
```

File: build_info/libcddb2.control [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+Maintainer: @DEB_MAINTAINER@
+Version: @DEB_LIBCDDB_V@
+Section: Libraries
+Priority: optional
+Description: library to access CDDB data
+ libcddb is a C library to access data about Audio CD on a CDDB server
+ .
+ It allows one to search the database for CD matches, get detailed information about a CD and submit new CD entries to the database. It supports both the custom CDDB protocol and the tunneling of the query over HTTP. HTTP Proxying is supported. It also features a cache system.
+ .
+ This package contains the libcddb runtime library.
```
Comment (TheRealKeto) [inline build_info/libcddb2.control:12]: Stop copypasting Debian's control file

```suggestion
Description: library to access CDDB data.
 libcddb is a C library to access data about Audio CD
 on a CDDB server. It allows one to search the database for
 CD matches, get detailed information about a CD and submit
 new CD entries to the database. It supports both the custom
 CDDB protocol and the tunneling of the query over HTTP. HTTP
 Proxying is supported. It also features a cache system. This
 package contains the libcddb runtime library.
```

File: build_info/libcddb2-dev.control [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+Depends: libcddb2 (= @DEB_LIBCDDB_V@)
+Provides: libcddb-dev
+Section: Development
+Priority: optional
+Description: development files for libcddb
+ libcddb is a C library to access data about Audio CD on a CDDB server
+ .
+ It allows one to search the database for CD matches, get detailed information about a CD and submit new CD entries to the database. It supports both the custom CDDB protocol and the tunneling of the query over HTTP. HTTP Proxying is supported. It also features a cache system.
+ .
+ This package contains the header files and static library needed to compile applications that use libcddb.
```
Comment (TheRealKeto) [inline build_info/libcddb2-dev.control:14]: Last control file, finally.

```suggestion
Description: development files for libcddb.
 libcddb is a C library to access data about Audio CD
 on a CDDB server. It allows one to search the database
 for CD matches, get detailed information about a CD and
 submit new CD entries to the database. It supports both
 the custom CDDB protocol and the tunneling of the query
 over HTTP. HTTP Proxying is supported. It also features a
 cache system. This package contains the header files and
 static library needed to compile applications that use
 libcddb.
```

**Review by TheRealKeto — CHANGES_REQUESTED** [review]
This took me a long time for some reason...

**Review by CRKatri — APPROVED** [review]

**Pull request merged**